src/components/Homepage/CategoryGrid.vue

```diff
@@ -0,0 +1,190 @@
+<template>
+	<div class="row text-center frame">
+		<h1 class="columns small-12 impact-text heading-text">
+			Browse loans by category
+		</h1>
+		<div class="row card-row">
+			<router-link
+				:to="loanChannel.url"
+				v-for="loanChannel in loanChannels" :key="loanChannel.id"
+				class="small-6 medium-4"
+				v-kv-track-event="[
+					'homepage',
+					'click-Category-card',
+					`${loanChannel.name}-category-card`,
+					'true']"
+			>
+				<div class="category-card-overlay">
+					<kv-responsive-image
+						:images="[
+							['small', loanChannel.image.url],
+							['small-retina', loanChannel.retinaImage.url]
+						]"
+						class="loan-channel-image"
+					/>
+					<div class="category-card-title-wrap">
+						<p class="category-card-title featured-text">
+							{{ loanChannel.name }}
+						</p>
+					</div>
+				</div>
+			</router-link>
+			<router-link
+				:to="`/lend-by-category`"
+				class="small-12 medium-4"
+				v-kv-track-event="[
+					'homepage',
+					'click-View all categories',
+					'view-all-categories-homepage-card',
+					'true']"
+			>
+				<div class="solid-card-overlay">
+					<div class="view-all-card">
+						<div class="solid-category-card-title-wrap">
+							<p class="category-card-title featured-text view-all-card-title">
+								View all
+							</p>
+						</div>
+					</div>
+				</div>
+			</router-link>
+		</div>
+		<div class="featured-text row small-10 category-sub-text">
+			By lending as little as $25, you can help people around the world
+			create opportunity for themselves and their communities.<br>
+			<router-link
+				:to="`/lend-by-category`"
+				class="get-started-link"
+				v-kv-track-event="[
+					'homepage',
+					'click-Get started',
+					'lend-by-category-homepage-redirect',
+					'true'
+				]"
+			>
+				Get started
+			</router-link>
+		</div>
+	</div>
+</template>
+
+<script>
+import _get from 'lodash/get';
+import categoryRowsQuery from '@/graphql/query/categoryRows.graphql';
+import KvResponsiveImage from '@/components/Kv/KvResponsiveImage';
+
+export default {
+	components: {
+		KvResponsiveImage,
+	},
+	inject: ['apollo'],
+	apollo: {
+		query: categoryRowsQuery,
+		preFetch: true,
+		result({ data }) {
+			this.loanChannels = _get(data, 'lend.loanChannels.values');
+		}
+	},
+};
+</script>
+
+<style lang="scss" scoped>
+@import 'settings';
+
+.heading-text {
+	padding: 3rem;
+	font-weight: 300;
+}
+
+.get-started-link {
+	display: contents;
```

Comment:
`contents` doesn't have good browser support yet, and I'm not sure if it's needed for this element since it doesn't have any children